src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php

```diff
@@ -141,7 +141,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             }
 
             if (!$fs->exists($warmupDir.'/'.$containerDir)) {
-                $fs->rename($realCacheDir.'/'.$containerDir, $warmupDir.'/'.$containerDir);
+                $fs->rename($realBuildDir.'/'.$containerDir, $warmupDir.'/'.$containerDir);
```

Comment:
If we have a separate buildDir and warming the cache. The container will never be located in $warmupDir. It will always be in `$readBuildDir`.

If we dont have a separate buildDir. Then $readBuildDir === $readCacheDir`